changelog/15429.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secrets/pki: pki/sign-verbatim uses not_before_duration defined role
```

Comment:
This is a bit terse perhaps?

Reply:
Hello, how about "Add support for pki/sign-verbatim use the not_before_duration field defined in the role"?

Reply:
Not to nitpick this to death, but maybe `Have pki/sign-verbatim use the ...`? This isn't conditional is why I suggest that.

Reply:
Makes sense.